WickedEngine/wiShaderCompiler.cpp

```diff
@@ -595,19 +457,6 @@ namespace wiShaderCompiler
 			}
 		}
 #endif // SHADERCOMPILER_ENABLED_DXCOMPILER
-
-#ifdef SHADERCOMPILER_ENABLED_D3DCOMPILER
```

Comment:
Same, just keep the d3dcompiler code parts and keep them disabled. It's good if we need to test something that we can just re enable it.